apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -56,41 +74,86 @@ TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a t
+| `setVolume`        | `(volumeValOrFn: SetStateAction<number>) => void`  | Sets the volume of the video.                                                                  |
+| `mute`             | `() => void`                                       | Mutes the video.                                                                               |
+| `unmute`           | `() => void`                                       | Unmutes the video.                                                                             |
+| `toggleMute`       | `() => void`                                       | Toggles between muting and unmuting the video.                                                 |
+| `setDuration`      | `(duration: number) => void`                       | Sets the duration of the video.                                                                |
+| `seek`             | `(time: number) => void`                           | Seeks the video to a specific time in seconds.                                                 |
+| `seekToPercent`    | `(percent: number) => void`                        | Seeks the video to a specific percentage of its duration.                                      |
+| `seekRelative`     | `(secondsValOrFn: SetStateAction<number>) => void` | Seeks the video forward or backward by a given number of seconds.                              |
+| `enableCC`         | `() => void`                                       | Enables closed captions.                                                                       |
+| `disableCC`        | `() => void`                                       | Disables closed captions.                                                                      |
+| `toggleCC`         | `() => void`                                       | Toggles closed captions on and off.                                                            |
+| `enterFullScreen`  | `() => void`                                       | Enters fullscreen mode.                                                                        |
+| `exitFullScreen`   | `() => void`                                       | Exits fullscreen mode.                                                                         |
+| `toggleFullScreen` | `() => void`                                       | Toggles between entering and exiting fullscreen mode.                                          |
+
+<h2>Keyboard shortcuts</h2>
+
+The `VideoPlayer` component supports the following keyboard shortcuts for controlling video playback:
+
+- `Space` or `k`: Toggles between playing and pausing the video.
+- `m`: Toggles between muting and unmuting the video.
+- `f`: Toggles between entering and exiting fullscreen mode.
+- `c`: Toggles closed captions on and off.
+- `ArrowLeft`: Seeks the video backward by 5 seconds.
+- `ArrowRight: Seeks the video forward by 5 seconds.
```

Comment:
```suggestion
- `ArrowRight`: Seeks the video forward by 5 seconds.
```

Reply:
Great spot, thanks!

Reply:
Resolved in 248ae2817b68adcdc9f2f920decfe7a88c37999c